packages/jest-environment/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@jest/environment",
```

Comment:
Mostly so I could reuse `jest-mock` types
 It'll be useful for all test environments, though

Reply:
/cc @lirbank relevant for your migrations